DEVELOPING.md

```diff
@@ -23,21 +25,24 @@ can build the styleguide with `make styleguide` which will create HTML files in
 
 ### Updating GH Pages distribution
 
-We use [Github Pages][] to distribute the compiled Javascript code, stylesheets, and
-the styleguide from the `./docs` folder.
+We use [Github Pages][] to distribute the styleguide from the `./docs` folder.
+
+If you make any changes to the stylesheets run `make build-pages` to
+update the `./docs` directory and add it to your commit.
 
-If you make any changes to the code or stylesheets run `make build-pages` to
-update the `./docs` directory.
+For backwards compatibility we also distribute the code and styles for `v2.1.0`
+from Github Pages. These are not updated automatically anymore since we use
+[`unpkg.com`][unpkg].
 
 [Github Pages]: https://help.github.com/categories/github-pages-basics/
+[unpkg]: https://unpkg.com
 
 ### Releasing a new version of the SDK
 
 1. Add new release to changelog by renaming “Upcoming” to “v1.2.3”.
 2. Update API documentation for added methods to include the version they were
    added in (e.g. _“Since 1.2.3”_)
 3. Update the version in `package.json`.
-4. [Update GH Pages distribution](#updating-gh-pages-distribution)
-5. Commit to master.
-6. Tag commit as `vX.Y.Z` and push tag to GitHub
-7. Publish new version of package on `npm`
+4. Commit to master.
+5. Tag commit as `vX.Y.Z` and push tag to GitHub
```

Comment:
Why are we tagging manually here, we can use semantic-release-cli to do this, we have that implemented in [contentful.js](https://github.com/contentful/contenful.js) the only thing is we should use conventional log commit messages, this way we'll have even release notes automatically generated